model/reader.go

```diff
@@ -507,6 +507,61 @@ func (r *PdfReader) GetOutlines() (*Outline, error) {
 	return outline, nil
 }
 
+// RepairAcroForm attempts to rebuild the AcroForm fields using the widget
+// annotations present in the document pages.
+func (r *PdfReader) RepairAcroForm() error {
```

Comment:
would it make sense to include some repair options so that in the future we can add some optional parameters?  One example could be including only some fields to be included, or partial page list and such.

Reply:
Yes, adding an empty `AcroFormRepairOptions` struct would be a good idea as it leaves room for future configuration. I'm not sure what the options would be though. Typically, a user would call this method if the AcroForm is broken or missing, and it needs rebuilding. In these cases, probably all fields would need to be included.

Reply:
OK that sounds good.  Yes let´s add it to allow future improvements.